src/core/curves.js

```diff
@@ -63,6 +63,16 @@ var curveDetail = 20;
  *bezier(250,250,0, 100,100,0, 100,0,0, 0,100,0);
  * </code>
  * </div>
+ *@alt
```

Comment:
this one actually needs to be attached to the first bezier block above. i've moved it here:
https://github.com/processing/p5.js/commit/80f003001ec1304dbf49713cc68ad8a86fe143d4